### ISSUE TITLE
Support multiple metrics when running alpha, beta, etc.

### ISSUE THREAD

**Issue by thermokarst** [qiime2/q2-diversity#146]
This is blocked by interfaces not supporting `Set` (e.g. [q2cli](https://github.com/qiime2/q2cli/issues/159)).

This recently came up [on the forum](https://forum.qiime2.org/t/parameters-file/1308).

**Comment by ChrisKeefe** [comment]
@thermokarst, this one can be closed now, right? 

![image](https://user-images.githubusercontent.com/39198770/90301108-b09d3a00-de52-11ea-8e39-7248bfc02d76.png)

**Comment by thermokarst** [comment]
No, still blocked. We need optional/variadic outputs to handle alpha and beta (note the issue is not limited to alpha-rarefaction).

**Comment by ChrisKeefe** [comment]
Thanks, that makes sense - alpha-rarefaction slips through because it _always produces one viz_.
